packages/babel-cli/src/babel/dir.js

```diff
@@ -43,18 +43,24 @@ module.exports = function (commander, filenames) {
     }
   }
 
+  let filterDotFiles = function (x) {
+    return x[0] === ".";
+  };
```

Comment:
Edit: `filterDotFiles` is actually the default value, we could just pass `null`.